code/datums/martial/krav_maga.dm

```diff
@@ -92,40 +48,104 @@
 	D.visible_message("<span class='warning'>[A] leg sweeps [D]!</span>", \
 					  	"<span class='userdanger'>[A] leg sweeps you!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_kick.ogg', 50, 1, -1)
-	D.apply_damage(5, BRUTE)
+	D.apply_damage(5, BRUTE, A.zone_selected)
 	D.Paralyze(40)
 	log_combat(A, D, "leg sweeped")
 	return 1
 
-/datum/martial_art/krav_maga/proc/quick_choke(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)//is actually lung punch
+/datum/martial_art/krav_maga/proc/lung_punch(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
 	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
 				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
 	if(D.losebreath <= 10)
 		D.losebreath = CLAMP(D.losebreath + 5, 0, 10)
 	D.adjustOxyLoss(10)
-	log_combat(A, D, "quickchoked")
+	log_combat(A, D, "lung punched")
 	return 1
 
 /datum/martial_art/krav_maga/proc/neck_chop(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
 	D.visible_message("<span class='warning'>[A] karate chops [D]'s neck!</span>", \
 				  	"<span class='userdanger'>[A] karate chops your neck, rendering you unable to speak!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.apply_damage(5, A.dna.species.attack_type)
+	D.apply_damage(5, A.dna.species.attack_type, A.zone_selected)
 	if(D.silent <= 10)
 		D.silent = CLAMP(D.silent + 10, 0, 10)
 	log_combat(A, D, "neck chopped")
 	return 1
 
+/datum/martial_art/krav_maga/proc/eye_strike(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	D.visible_message("<span class='warning'>[A] eye struck [D]!</span>", \
+				  	"<span class='userdanger'>[A] strikes your eye!</span>")
+	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	D.apply_damage(3, BRUTE, A.zone_selected)
+	D.adjust_blurriness(3)
+	D.adjust_eye_damage(rand(1,3))
+	var/obj/item/organ/eyes/eyes = D.getorganslot(ORGAN_SLOT_EYES)
+	if (!eyes)
+		return
+	if(eyes.eye_damage >= 10)
+		D.adjust_blurriness(15)
+		if(D.stat != DEAD)
+			to_chat(D, "<span class='danger'>Your eyes start to bleed profusely!</span>")
+		to_chat(D, "<span class='danger'>You become nearsighted!</span>")
+		D.become_nearsighted(EYE_DAMAGE)
+		if(prob(50))
+			if(D.stat != DEAD)
+				if(D.drop_all_held_items())
+					to_chat(D, "<span class='danger'>You drop what you're holding and clutch at your eyes!</span>")
+			D.adjust_blurriness(10)
+			D.Unconscious(20)
+			D.Paralyze(40)
+		if (prob(eyes.eye_damage - 10 + 1))
+			D.become_blind(EYE_DAMAGE)
+			to_chat(D, "<span class='danger'>You go blind!</span>")
```

Comment:
eyestabbing is already a proc, i think this is all copypasted

Reply:
eyestabbing can only be called by items, im workin on something that would merge eyestabbing and other means of eye attacks

Reply:
this is really easy, just make eyestabbing a proc on the eyes.

me not accept copypasta